modules/ckanext-sixodp_showcase/ckanext/sixodp_showcase/templates/sixodp_showcase/search.html

```diff
@@ -7,15 +7,9 @@
   <li class="active">{{ h.nav_link(_('Showcases'), controller='ckanext.sixodp_showcase.controller:Sixodp_ShowcaseController', action='search', highlight_actions = 'new index') }}</li>
 {% endblock %}
 
+
 {% block pre_primary %}
   {% set featured_showcases = h.get_featured_showcases() %}
-  {#
-  {% if featured_showcases %}
-    <div class="page-highlight">
-      {{ h.snippet('snippets/horizontal_accordion.html', items=featured_showcases, image_identifier="featured_image_display_url", meta_identifier="category", controller='ckanext.sixodp_showcase.controller:Sixodp_ShowcaseController') }}
-    </div>
-  {% endif %}
-  #}
 {% endblock %}
```

Comment:
I would not remove this, at least until someone has decided if we want featured showcases. If we don't, they need to be removed for showcase datamodel, if we do, this needs to be fixed to be working template.

Reply:
Ok readded it